webview/src/plots/components/styles.module.scss

```diff
@@ -268,12 +268,28 @@ $gap: 20px;
 
 .dropSectionMaximized,
 .dropTarget {
+  position: relative;
   height: auto;
-  opacity: 0.5;
-  border: 3px dashed $accent-color;
+  width: 100%;
   display: flex;
   justify-content: center;
   align-items: center;
+
+  & > th {
+    display: block;
+    width: 100%;
+  }
```

Comment:
Does this still apply to anything? It seems like the drop target was turned into a `div` in another part of the code, and changing a `th`'s `display` seems like it would cause a lot more chaos if it were applying.

Reply:
Good catch. It is applying because the current implementation re-renders the dragged cell as the drop target, which in this case is a `th`. The `display: block` is indeed unnecessary, but the `width: 100%` is there so the cell expands to fill the drop target.